.github/workflows/linux-ci-helper.sh

```diff
@@ -148,16 +171,16 @@ fi
 # Update packages (ex. apt-get update -y -qq)
 #
 echo "${PRGNAME} [INFO] Updates."
-${PACKAGE_MANAGER_BIN} ${PACKAGE_UPDATE_OPTIONS}
+/bin/sh -c "${PACKAGE_MANAGER_BIN} ${PACKAGE_UPDATE_OPTIONS}"
```

Comment:
This is a weird change -- why should we invoke a shell to invoke a program?

Reply:
There was no error(by shellcheck) in this line, but I fixed it in the same way as the error( [SC2086](https://github.com/koalaman/shellcheck/wiki/Sc2086) ) for the `${INSTALL_PACKAGES}` variable a few lines down(L:180).
Most of shellcheck's suggestions are variable quotes, so I've fixed this line as well.

If it is not needed to run another shell, I can undo it.
Please tell me your opinion.